src/ros2/usb_cam_node.cpp

```diff
@@ -423,7 +425,7 @@ void UsbCamNode::update()
     // If the camera exposure longer higher than the framerate period
     // then that caps the framerate.
     // auto t0 = now();
-    bool isSuccessful = (m_parameters.pixel_format_name == "mjpeg") ?
+    bool isSuccessful = (m_parameters.pixel_format_name == "raw_mjpeg") ?
```

Comment:
@hilookas should probably support both `mjpeg` and `raw_mjpeg`, right?

Reply:
To my understand, mode mjpeg and mode raw_mjpeg is different.

mjpeg mode should decode the mjpeg image from driver, and send it to m_image topic, which will be compressed again to /compressed by image_transport

raw_mjpeg should send jpeg encoded image from driver and send it directly to /compressed, m_image topic should keep empty preventing cpu-consuming jpeg decoding. This mode should be optimized to send compressed image only.

Reply:
> mjpeg mode should decode the mjpeg image from driver, and send it to m_image topic, which will be compressed again to /compressed by image_transport

And infact there is no "mjpeg" mode right now for ros2